src/always-be-playing.js

```diff
@@ -135,6 +121,71 @@ export default class GapSkipper {
   }
 
   /**
+   * Handler for `waiting` events from the player
+   *
+   * @private
+   */
+  waiting_() {
```

Comment:
It could be a relic of some old changes, but I remembered seeing waiting fired twice. I will switch it out and see if I can uncover why that was happening instead.